README.md

```diff
@@ -1,4 +1,4 @@
-# AngularFire2
+# AngularFire3
```

Comment:
Got ahead of myself :wink:

There's a vim key that increments numbers. I guess I hit it by accident...